libs/wcm/context/connectionProvider.test.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useContext } from 'react';
+import { mount } from 'enzyme';
+import ConnectionProvider from './connectionProvider';
+import ConnectionContext from './connectionContext';
+import { EVENTS } from '../constants/lifeCycle';
+
+describe('connectionProvider', () => {
+  it('Should mount a child with access to the default context', () => {
+    const TestComponent = () => {
+      const { session } = useContext(ConnectionContext);
+      return (
+        <div>
+          {Object.keys(session).map(key => (<span key={key}>{key}</span>))}
+        </div>
+      );
+    };
+    const wrapper = mount(
+      <ConnectionProvider>
+        <TestComponent />
+      </ConnectionProvider>,
+    );
+    expect(wrapper.find('span').at(0).text()).toBe('request');
+    expect(wrapper.find('span').at(1).text()).toBe('data');
+    expect(wrapper.find('span').at(2).text()).toBe('loaded');
+  });
+
+  it('Should update the context data using setters', () => {
+    const TestComponent = () => {
+      const {
+        events, pushEvent, setPairings, pairings,
+      } = useContext(ConnectionContext);
+      useEffect(() => {
+        pushEvent([{ name: EVENTS.SESSION_DELETE, meta: { session: { id: '1' } } }]);
```

Comment:
pushEvent should be an object

Reply:
Push event is a function. As you can see it is "called".

Reply:
Done